packages/block-library/src/paragraph/index.js

```diff
@@ -201,20 +197,15 @@ export const settings = {
 			migrate( attributes ) {
 				return {
 					...attributes,
-					content: [
-						<RawHTML key="html">{ attributes.content }</RawHTML>,
-					],
+					content: create( attributes.content ),
```

Comment:
Thanks, I'll look into it.

Reply:
So before we were using React as an HTML parser, and our rich text structure would be a React structure, so this was good. Now the rich text structure uses the browser to parse the HTML. This is actually an interesting thought, because we might be able to use `dangerouslySetInnerHTML` as an the HTML parser instead of the browser. But does React Native have something similar? I don't know how it works there behind the scenes.

Reply:
I don't see anything there, so my guess is that this doesn't work either for master on mobile: https://github.com/facebook/react-native/search?q=dangerouslySetInnerHTML&type=Code.

Reply:
Looks like we'd have to avoid the use of `<RawHTML />` as well for mobile.

Reply:
At the moment we use the following to pass HTML to Aztec component:
https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/rich-text/index.native.js#L159

I don't think `RawHTML` is ever used in RN context.